graphql/certificate_of_conduct/mutations.ts

```diff
@@ -0,0 +1,44 @@
+import {Arg, Authorized, Mutation, Resolver} from "type-graphql";
+import * as GraphQLModel from "../generated/models";
+import {Role} from "../authorizations";
+
+import {deactivateStudent} from "../../common/student/activation";
+import {getStudent} from "../util";
+import * as CertificateOfConduct from "../../common/certificate-of-conduct/certificateOfConduct";
+import {updateCertificateOfConduct} from "../../common/student";
+import {getLogger} from "log4js";
+const logger = getLogger("Certificate of Conduct");
+
+
+@Resolver(of => GraphQLModel.Certificate_of_conduct)
+export class MutateCertificateOfConductResolver {
+    @Mutation(returns => Boolean)
+    @Authorized(Role.ADMIN)
+    // eslint-disable-next-line camelcase
+    async updateCertificate(@Arg("studentId") studentId: number,
```

Comment:
Unfortunately mutations live in a global namespace, thus "updateCertificate" is a bit too generic as we have 3 kinds of certificates (and counting :)) ... Maybe "certificateOfConductUpdate" would be a better name (although quite long) ...